instrumentation/redisson-3.0/src/test/groovy/RedissonClientTest.groovy

```diff
@@ -47,8 +47,14 @@ class RedissonClientTest extends AgentTestRunner {
     .port(port).build()
   @Shared
   RedissonClient redisson
+  @Shared
+  String address = "localhost:" + port
 
   def setupSpec() {
+    if ("true".equals(System.properties.getProperty("testLatestDeps"))) {
```

Comment:
You can use `java.lang.Boolean#getBoolean`

Reply:
👍 thanks.